docs/src/components/main-nav.tsx

```diff
@@ -30,9 +30,9 @@ export function MainNav(props: { items: NavItem[] }) {
                   key={index}
                   href={item.href}
                   className={cn(
-                    "text-muted-foreground flex items-center justify-center rounded-md text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:opacity-50 disabled:pointer-events-none ring-offset-background hover:bg-accent hover:text-accent-foreground h-9 px-3",
```

Comment:
definetely wanna keep the focus states here to make sure it's keyboard friendly

Reply:
okay, I'll add them right away

Reply:
i have added back the focus states